src/components/AdvancedSettingsDialog.tsx

```diff
@@ -78,8 +78,15 @@ export function AdvancedSettingsDialog() {
 
   const receiver = useWatch({ control, name: "receiver" });
 
+  const handleOpenChange = (isOpen: boolean) => {
+    if (!isOpen) {
+      reset(advancedSettings);
```

Comment:
🤔 why do we need this here?

Reply:
CoW Feedback:

> "When I update settings fields, then close the modal without pressing on the Save button, the values I've changes are remain to be displayed in the fields. 
However, the changes are not applied when place an order. This moght be confising to a user: settings show different info than order review modal. 
So here I can propose 2 options: 
> 1. Remove 'Save' button and auto-save all changes made on the Settings modal
> 2. OR reset changes to defaults when close settings modal without pressing on the Save button"

This is to implement the second solution.